src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
Avoid using the 'body' tag for styling. It's recommended to style all elements using `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`. So you can add a class to the body and use it for styling.